src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -171,6 +171,7 @@ export class DotEditLayoutComponent implements OnInit {
         this.isModelUpdated = false;
         this.form.valueChanges.subscribe(() => {
             this.isModelUpdated = !_.isEqual(this.form.value, this.initialFormValue.value);
+            // TODO: Set sidebar to null if sidebar location is empty
```

Comment:
Add more explaining here, something like "we're expecting a change in the backend bla bla bla"